### PR TITLE
Feature/adding api for epicure data

### PR DESCRIPTION
**Pull request by Eran-Moveo**
correcting the code

### CODE REVIEW

**Comment by Eran-Moveo** [comment]
@DanielKissos

**Comment by Eran-Moveo** [comment]
@DanielKissos

File: src/db/epicureDB.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    useUnifiedTopology: true,
+    useCreateIndex: true,
+    useFindAndModify: false,
+    autoIndex: false, // Don't build indexes
+    poolSize: 10, // Maintain up to 10 socket connections
+    serverSelectionTimeoutMS: 5000, // Keep trying to send operations for 5 seconds
+    socketTimeoutMS: 45000, // Close sockets after 45 seconds of inactivity
+    family: 4 // Use IPv4, skip trying IPv6
+  };
+const uri = 'mongodb+srv://admin:LMRI8mDvKMJlRjwk@cluster1.p467e.mongodb.net/<Epicure>?retryWrites=true&w=majority';
```
Comment (kennethmoveo) [inline src/db/epicureDB.js:16]: All credentials & passwords need to be in config file that you don't upload to github

**Review by kennethmoveo — APPROVED** [review]
Great work! I've left one comment but the PR is approved

**Pull request merged**